bin/commandTopicConsumer.py

```diff
@@ -0,0 +1,165 @@
+#!/usr/bin/env python3
+#
+# Copyright 2020 Confluent Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+#
+# Dump ksqlDB persistent queries to stdout.
+#
+# Usage:
+#  commandTopicConsumer.py -f <client-conf-file>
+#
+
+from confluent_kafka import KafkaError, KafkaException, version
+from confluent_kafka import Producer, Consumer
+import json
+import logging
+import argparse
+import uuid
+import sys
+import re
+
+
+class CommandRecord (object):
+    def __init__(self, stmt):
+        self.stmt = stmt
+
+    def __str__(self):
+        return "({})".format(self.stmt)
+
+    @classmethod
+    def deserialize(cls, binstr):
+        d = json.loads(binstr)
+        return CommandRecord(d['statement'])
+
+max_offset = -1001
+def consumer_latest_offsets(consumer, partitions):
+    global max_offset
+    for p in partitions:
+        high_water = consumer.get_watermark_offsets(p)[1]
+        if high_water >= max_offset:
+            max_offset = high_water
+
+class CommandConsumer(object):
+    def __init__(self, ksqlServiceId, conf):
+        self.consumer = Consumer(conf)
+        self.topic = '_confluent-ksql-{}_command_topic'.format(ksqlServiceId)
+
+    def consumer_run(self):
+        global max_offset
+        self.consumer.subscribe([self.topic], on_assign=consumer_latest_offsets)
+        #print("max offset = ", max_offset)
```

Comment:
We should clearly display to the user which offset in the command topic we are reading up to, IMO.